src/Infrastructure.EntityFramework/Repositories/DatabaseContext.cs

```diff
@@ -141,7 +135,9 @@ protected override void OnModelCreating(ModelBuilder builder)
             eUser.ToTable(nameof(User));
             eOrganizationApiKey.ToTable(nameof(OrganizationApiKey));
             eOrganizationConnection.ToTable(nameof(OrganizationConnection));
-            eSecret.ToTable(nameof(Secret));
+
+            // Apply all configurations specified in types implementing IEntityTypeConfiguration in the assembly.
```

Comment:
This comment is slightly reduncant, lets try and make it more generic.

I believe we should move this to the top of this method, and add a note that the other way is deprecated.

```c#
// Scans and loads all configurations implementing the `IEntityTypeConfiguration` from the
//  `Infrastructure.EntityFramework` Module. Note to get the assembly we can use a random class
//   from this module.
```

Reply:
resolved in https://github.com/bitwarden/server/pull/2144/commits/9c410672d8b381974f1ab20a384028eb21fab8ed